src/Tokenizer/Token.php

```diff
@@ -474,6 +474,16 @@ public function isWhitespace($whitespaces = " \t\n\r\0\x0B")
     }
 
     /**
+     * Check if token is a logical operator.
+     *
+     * @return bool
+     */
+    public function isLogicalOperator()
```

Comment:
Technically, adding this method is a new feature which would require a minor version increment. But this PR is a bugfix that should target a lower branch. Could you please remove it?